compiler/rustc_mir_build/src/thir/pattern/check_match.rs

```diff
@@ -479,12 +479,30 @@ impl<'p, 'tcx> MatchVisitor<'_, 'p, 'tcx> {
             AdtDefinedHere { adt_def_span, ty, variants }
         };
 
+        // Emit an extra note if the first uncovered witness is
+        // visibly uninhabited anywhere in the current crate.
```

Comment:
It seems like you changed the logic, could you update this comment to match the new logic?